packaging/hudi-flink-bundle/pom.xml

```diff
@@ -162,6 +171,22 @@
                   <pattern>org.apache.avro.</pattern>
                   <shadedPattern>${flink.bundle.shade.prefix}org.apache.avro.</shadedPattern>
                 </relocation>
+                <relocation>
+                  <pattern>org.apache.commons.io.</pattern>
```

Comment:
this reminds me. older bootstrap index files may have an unshaded key comparator class saved within the HFile. Does that cause any issues? ie can we read such files?

Reply:
I need to check that.  This is another compatibility test.

Reply:
Actually, bootstrap index HFiles always use our own comparator class under `org.apache.hudi` package so we're good here.  The problem is with the metadata table HFiles in Hudi 0.9.0.